templates/zerver/emails/email_base.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html lang="en">
+<head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <title>Zulip</title>
+        <style>
+
+        h3 {
+            font-family: Arial;
+            font-size: 30px;
+            margin: 5px 0px;
+            color: #555;
+        }
+
+        </style>
+</head>
+<body>
+<table width="80%" style="align:center; max-width:800px" align="center">
+    <tr>
+        <td>
+        {% block header %}{% endblock %}
```

Comment:
This should be indented. Also, the `<h3>` should go in here, rather than in the individual templates, though feel free to leave as is, since most of this is being completely replaced soon anyway.